lib/modules/manager/npm/extract/locked-versions.spec.ts

```diff
@@ -5,10 +5,13 @@ import { getLockedVersions } from './locked-versions';
 /** @type any */
 const npm = require('./npm');
 /** @type any */
+const pnpm = require('./pnpm');
 const yarn = require('./yarn');
+/** @type any */
```

Comment:
remove, does nothing in typescript .
it's also on wrong line, needs to be before the variable .


these requires should be converted to regular imports

Reply:
Removing only the comments for now. I will convert to regular imports in a separate PR as the tests need to be modified too.